packages/pds/src/phone-verification/plivo.ts

```diff
@@ -0,0 +1,124 @@
+import * as ui8 from 'uint8arrays'
+import { InvalidRequestError } from '@atproto/xrpc-server'
+import { plivoLogger as log } from '../logger'
+import Database from '../db'
+import { excluded } from '../db/util'
+
+type Opts = {
+  authId: string
+  authToken: string
+  appId: string
+}
+
+export class PlivoClient {
+  authId: string
+  appId: string
+  private authorization: string
+
+  constructor(public db: Database, opts: Opts) {
+    this.authId = opts.authId
+    this.appId = opts.appId
+    this.authorization =
+      'Basic ' +
+      ui8.toString(
+        ui8.fromString(`${opts.authId}:${opts.authToken}`, 'utf8'),
+        'base64pad',
+      )
+  }
+
+  async sendCode(phoneNumber: string) {
+    try {
+      const res = await this.makeReq(
+        `https://api.plivo.com/v1/Account/${this.authId}/Verify/Session/`,
```

Comment:
The trailing slash quirk could be worth a note.

Reply:
good call 👍